sklearn/mixture/bayesian_mixture.py

```diff
@@ -111,7 +118,14 @@ class BayesianGaussianMixture(BaseMixture):
         'kmeans' : responsibilities are initialized using kmeans.
         'random' : responsibilities are initialized randomly.
 
-    dirichlet_concentration_prior : float | None, optional.
+    weight_concentration_prior_type : {'dirichlet_process',
+            'dirichlet_distribution'}, defaults to 'full'.
```

Comment:
I don't know how to fix the first problem because if I put it on a unique line pep8 will not be happy.